samples/bluetooth/llpm/src/main.c

```diff
@@ -381,7 +381,15 @@ static void test_run(void)
 			printk("Latency failed (err %d)\n", err);
 		}
 
-		k_sleep(K_MSEC(200)); /* sleep 200 ms*/
+		k_sleep(K_MSEC(200)); /* wait for latency response */
+
+		if (llpm_latency.latency)
+		{
+			printk("Transmission Latency: %u (us), CRC errors %u\n",
+			       llpm_latency.latency, llpm_latency.crc_errors);
+		}
+
```

Comment:
If you don't receive anything within 200ms, you might want to print something?

```suggestion
		{
			printk("Did not receive a latency response\n");
		}
```